addons/mrp_account/tests/test_analytic_account.py

```diff
@@ -53,6 +53,14 @@ def setUpClass(cls):
 
 
 class TestAnalyticAccount(TestMrpAnalyticAccount):
+    def _set_move_picked(self, mo_form, val):
+        """Changing the qty_producing will not mark the picked automatically,
+        we need do it manually
+        """
+        with mo_form.move_raw_ids.edit(0) as move:
+            move.picked = val
```

Comment:
This method is too specific, it only ever does it for the first move in the MO. It's better to manually put the logic in each test that you're calling it in and drop this helper method